feat(components): add alert component (#1085)
swisspost/design-system: packages/documentation/src/stories/components/alert/alert.docs.mdx

      <Canvas of={AlertStories.ActionButtons} />

      ### Banner

      To present an alert as a banner anchored at the bottom of the page, simply add the class `.alert-fixed-bottom`.
      **These alerts are typically designed to be dismissible, therefore it is advisable to employ a web component for their implementation.**

      <Canvas of={AlertStories.Fixed} />
    </PostTabPanel>

    <PostTabHeader slot="tabs" panel="webComponent">Web Component</PostTabHeader>
    <PostTabPanel name="webComponent">
      <Canvas of={PostAlertStories.Default} />

      <Controls of={PostAlertStories.Default} />

      ## Installation

      The `<post-alert/>` element is part of the `@swisspost/design-system-components` package.
      For more information, read the <LinkTo kind="getting-started-components" story="docs">getting started with components guide</LinkTo>.

      ## Examples

      ### Contents

      Alerts can contain various HTML elements like paragraphs, lists, icons and dividers.

      By default all children of the `<post-alert/>` are placed in the body.
      Use the `heading` slot to place a child in the heading, and the `actions` slot for action buttons.

>       Learn more in <a rel="noopener" target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_templates_and_slots#adding_flexibility_with_slots">Mozilla's slots documentation</a>.

Reviewer: ```suggestion
      Learn more about<a rel="noopener" target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_templates_and_slots#adding_flexibility_with_slots">slot on mdn web docs</a>.
```
